scripts/langsmith_eval.py

```
@@ -0,0 +1,469 @@
+        )
+    return wrapped
+
+
+async def run_evaluation(
+    client: Client,
+    evaluator: MultiAgentEvaluator,
+    dataset_name: str,
+    evaluators: List[Any],
+    experiment_prefix: str = "sujbot2-qa-eval",
+    max_concurrency: int = 1,  # Use 1 to avoid async/DB pool conflicts
+    limit: Optional[int] = None,
```

Ruff (0.14.7)

Unused function argument: `limit`

(ARG001)